src/styles/app.css

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+.App {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 5%;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #f4f4f4;
+}
+
+.logo {
+  height: 10%;
+  width: 20%;
```

Comment:
Setting a static width for all screen sizes like this isnt the best idea, as it will look very small on mobile devices. A good approach for width is to set boundaries using properties like `min-width` and `max-width` as well as the standard `width` property set to 100%, something like:

```
min-width: 320px;
max-width: 800px;
width: 100%;
```
This means you stay within a given range and the image is responsive